src/System.Net.Primitives/src/System/Net/SocketException.netstandard17.cs

```diff
@@ -15,8 +15,10 @@ protected SocketException(SerializationInfo serializationInfo, StreamingContext
         {
             if (GlobalLog.IsEnabled)
             {
-                GlobalLog.Print("SocketException::.ctor(serialized) " + NativeErrorCode.ToString() + ":" + Message);
+                GlobalLog.Print($"SocketException::.ctor(serialized, SocketErrorCode={SocketErrorCode}):{Message}");
```

Comment:
Nit: would you mind reverting this line?  I'm currently overhauling the tracing done in the System.Net libs, and this will conflict.